packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -30,7 +30,8 @@ export default async function getExpoPushTokenAsync(options: Options = {}): Prom
 
   const deviceId = options.deviceId || (await getDeviceIdAsync());
 
-  const experienceId = options.experienceId || (Constants.manifest && Constants.manifest.id);
+  const experienceId =
+    options.experienceId || Constants.manifest?.currentFullName || Constants.manifest?.id;
```

Comment:
Just in case there are any differences introduced in `Constants.manifest?.currentFullName`, shouldn't we change this to 
``` 
options.experienceId || Constants.manifest?.id || Constants.manifest?.currentFullName;
```
so on the off chance there's a bug, we don't change existing users' push tokens

Reply:
Added my thoughts to the universe PR, linked below.